imbi/endpoints/project_ssm_configuration.py

```diff
@@ -152,6 +145,52 @@ async def post(self, *args, **kwargs) -> None:
                 else:
                     raise errors.InternalServerError(code)
 
+    async def delete(self, *args, **kwargs):
+        project_info = await self._get_project_info(kwargs['project_id'])
+        google_tokens = await self._get_google_tokens()
+        aws_session = aioboto3.Session()
+        body = self.get_request_body()
+
+        project_config = self.application.settings['project_configuration']
+        ssm_path_prefix = self.path_prefix(
+            project_config['ssm_prefix_template'],
+            project_info['project_slug'], project_info['project_type_slug'],
+            project_info['namespace_slug'])
+        name = ssm_path_prefix + body['name']
+
+        for environment in set(body['environments']):
+            role_arn = await self.get_role_arn(environment,
+                                               project_info['namespace_id'])
+            if not role_arn:
+                raise errors.Forbidden(
+                    'No role ARN found for namespace %d in %s',
+                    project_info['namespace_id'], environment)
+            creds = await self.get_aws_credentials(aws_session, role_arn,
+                                                   self.current_user,
+                                                   google_tokens.id_token,
+                                                   google_tokens.refresh_token)
+
+            try:
+                await aws.delete_parameter(aws_session, creds['access_key_id'],
+                                           creds['secret_access_key'],
+                                           creds['session_token'], name)
+            except botocore.exceptions.ClientError as error:
+                code = error.response['Error']['Code']
+                if code == 'ParameterNotFound':
+                    raise errors.ItemNotFound('ParameterNotFound')
+                else:
+                    raise errors.InternalServerError(code)
```

Comment:
The spec says that the HTTP status is 204. I think that tornado is going to default to a 200.
```suggestion
                    raise errors.InternalServerError(code)

            self.set_status(204)
```

Reply:
Good catch